src/components/ebay-area-chart/README.md

```diff
@@ -0,0 +1,46 @@
+<h1 style='display: flex; justify-content: space-between; align-items: center;'>
+    <span>
+        ebay-area-chart
+    </span>
+    <span style='font-weight: normal; font-size: medium; margin-bottom: -15px;'>
+        DS v3.7.0
+    </span>
+</h1>
+
+```marko
+<ebay-area-chart
+    title="Sample chart title"
+    description="Chart displaying random sample values"
+    series=[{
+        x: 1658818800000,
+        y: 1855,
+        label: '$1,855.00'
+    },{
+        x: 1658818900000,
+        y: 2200.04,
+        label: '$2,200.04'
+    },{
+        x: 1658819000000,
+        y: 1255,
+        label: '$1,255.00'
+    }]
+/>
+```
+
+## Description
+
+The area chart displays one to five series of data points as an interactive stacked area chart
+
+## Attributes
```

Comment:
Can we update this to match `button`.
All attributes and how to use are in `storybook`
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-button/README.md